.cascade-code/Chapman.edu/_components/left_nav/left_nav_children.vtl

```diff
@@ -0,0 +1,137 @@
+## Formerly LeftNavNoSiblings. Used for index pages.
+## New left nav format that works without a block
+
+#set ( $pagePath = $currentPage.path )
+#set ( $parentDir = $currentPage.parentFolder )
+
+## Builds the left navigation
+#macro ( buildLeftNav )
+    #buildLeftNavTitle
+    <ul>
+    #set ( $children = $parentDir.children )
+    #foreach ( $child in $children )
+        #set ( $isVisible = $child.metadata.getDynamicField('Hide from navigation').value != "Yes" )
```

Comment:
Ideally, everything in the loop could be encapsulated in a `buildLeftNavChild` macro.